lib/polyfill/media_capabilities.js

```diff
@@ -333,12 +333,34 @@ shaka.polyfill.MediaCapabilities = class {
     if (videoConfig.framerate) {
       displayType += `; framerate=${videoConfig.framerate}`;
     }
-    if (videoConfig.transferFunction === 'pq') {
-      // A "PQ" transfer function indicates this is an HDR-capable stream;
-      // "smpte2084" is the published standard. We need to inform the platform
-      // this query is specifically for HDR.
-      displayType += '; eotf=smpte2084';
+
+    // Based on internal sources.  Googlers, see go/cast-hdr-queries for source.
```

Comment:
I can't access that go link. It says It's "missing a resource key" and that you should go to go/missingresourcekey.

Reply:
Accessible now, good.